crates/kitsune2_showcase/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "kitsune2_showcase"
+version = "0.0.1-alpha"
+description = "kitsune2 p2p / dht showcase app"
+license = "Apache-2.0"
+homepage = "https://github.com/holochain/kitsune2"
+documentation = "https://docs.rs/kitsune2_showcase"
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
+keywords = ["holochain", "p2p", "dht", "networking"]
+categories = ["network-programming"]
+edition = "2021"
+
+[dependencies]
+clap = { workspace = true, features = ["derive", "wrap_help"] }
+kitsune2 = { workspace = true }
+nu-ansi-term = "0.50.0"
+reedline = { version = "0.38.0", features = ["external_printer"] }
```

Comment:
I'd love to standardize on the tools we use for CLIs around Holochain. I don't think `ratatui` is appropriate here but I wonder if there's anything we could share with the scaffolding here for interactive CLIs?

Reply:
A possibility would be https://github.com/holochain/scaffolding/blob/main/Cargo.toml#L29 which I think is what gives the up/down arrow menus and takes user input. That might work nicely as an interaction pattern for this app. 

But I'm fine with whatever gets picked here really, it's more of a general desire to be able to switch between the code for different CLIs and not have to go back to being a newbie with the library :D

Reply:
dialoguer looks good except I don't think it supports printing to the terminal while the prompt is active, so may not be viable for this use-case.